tests-wasm/y-array.tests.js

```diff
@@ -11,8 +11,8 @@ export const testInserts = tc => {
     t.compare(d1.id, 1)
     var x = d1.getArray('test');
 
-    d1.transact(txn => x.insert(txn, 0, [1, 2.5, 'hello', ['world'], true]))
-    d1.transact(txn => x.push(txn, [{key:'value'}]))
+    x.insert(0, [1, 2.5, 'hello', ['world'], true])
```

Comment:
Nice result of having implicit transactions is that we no longer need to build transaction scope by hand.